src/js/components/AppPageComponent.jsx

```diff
@@ -157,7 +157,16 @@ var AppPageComponent = React.createClass({
     });
   },
 
-  onScaleAppError: function (errorMessage) {
+  onScaleAppError: function (errorMessage, statusCode, instances) {
+    if (statusCode === 409) {
+      var appId = this.state.appId;
+      const dialogId = DialogActions.
+        confirm("App scaling failed.Do you want to try forcing?");
+      DialogStore.handleUserResponse(dialogId, function () {
+        AppsActions.scaleApp(appId, instances, true);
+      });
+      return;
```

Comment:
@Raffo Nice. Can you remove the ```return``` and make an if-else-block from this for better readability?
```javascript
if (statusCode === 409) {
  ...
} else {
  DialogActions.alert(`Not scaling: ${errorMessage.message || errorMessage}`);
}
```

@mlunoe 
The response from the user is handled asynchronously in  the ```DialogStore.handleUserResponse(dialogId, success-handler, dismiss-handler)``` defined here. No dismiss-handler is set here, so no further action is triggered on cancel (except of closing the dialog of course).